code/modules/virus2/effect/stage_1.dm

```diff
@@ -138,13 +138,14 @@
 	max_count = 10
 
 /datum/disease2/effect/radresist/activate(var/mob/living/carbon/mob)
-	if(mob.reagents.get_reagent_amount(HYRONALIN) < REAGENTS_OVERDOSE / 2)
+	var/datum/reagent/hyronalin/hyro
+	if(mob.reagents.get_reagent_amount(HYRONALIN) < hyro.overdose_am / 2)
```

Comment:
That is a `can't access null.overdose_am` runtime by the looks of it
You can get the hyronalin reagent with `reagents.get_reagent(HYRONALIN)` and then check the amount and overdose from there.